ports/boost-python/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "boost-python",
   "version-string": "1.75.0",
```

```suggestion
  "version-semver": "1.75.0",
```
This might be [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver) here.